### DISCUSSION TITLE
Prompt closes popup without performing actions

### DISCUSSION THREAD

**Discussion by kskarra** [getgauge/taiko#2001] [General]
Has there been any changes to the prompt functionality in version(s) after 1.1.0? 
Tests run fine for now, but when I try to update taiko to a newer version it fails on steps that include prompts.

The prompts themselves are being successfully handled by Taiko, but the actions aren't registered in the browser. Tried in both headless and non-headless mode.

Version: 1.2.4 (Chromium: 88.0.4314.0) RELEASE

**Comment by zabil** [comment]
There seems to be a regression on the prompt functionality as per https://github.com/getgauge/taiko/issues/1989

**Reply by kskarra** [reply]
Alright, thanks for letting me know quickly! :)
